doc/protocols/sigprefixlist/SigPrefixListProtocol.md

```diff
@@ -0,0 +1,74 @@
+# SIG Prefix List Protocol
+
+## Overview
+The protocol is designed to enable SIGs to automatically fetch IP prefix to IA mappings. It uses a distributed coordinated infrastructure with the following design and security goals. 
+
+The system should
+- Be resilient to hijacking attacks 
```

Comment:
Here and in the following: The system shouldn't just be "resilient" but "prevent" the corresponding attacks.

Reply:
Also, as this list is part of a sentence, so punctuation and capitalization should respect this. See my suggestion.

Reply:
I corrected that now according to your previous comment.